Doc/reference/expressions.rst

```diff
@@ -815,30 +815,44 @@ Subscriptions
    object: dictionary
    pair: sequence; item
 
-Subscription of a sequence (string, tuple or list) or mapping (dictionary)
-object usually selects an item from the collection:
+The subscription of an instance of a :ref:`container class <sequence-types>`
```

Comment:
Seems weird to link this to sequence types. Maybe just drop the link? The later explanation is clearer about sequences and mappings.
```suggestion
The subscription of an instance of a container class
```

Reply:
I think this section in the docs perhaps originally only discussed sequence types. Now, however, it discusses all types of container types -- the link takes us to here: https://docs.python.org/3/reference/datamodel.html#emulating-container-types